vignettes/MetricsWeighted.Rmd

```diff
@@ -248,5 +250,17 @@ perf$`Tweedie p` <- as.numeric(as.character(perf$`Tweedie p`))
 plot(`R-squared` ~ `Tweedie p`, data = perf, type = "s")
 ```
 
+### Murphy diagrams
+
+The same logic as in the last example can be used to create so-called *Murphy diagrams* [@gneiting]. The function `murphy_diagram()` wraps above calls and allows to get elementary scores for one or multiple models across a range of theta values.

```

Comment:
Just a thought: We could be nice and also link to https://cran.r-project.org/package=murphydiagram.